Assets/Scripts/Game/UserInterfaceWindows/DaggerfallGuildServicePopupWindow.cs

```diff
@@ -574,14 +574,18 @@ private void DonationMsgBox_OnGotUserInput(DaggerfallInputMessageBox sender, str
             {
                 if (playerEntity.GetGoldAmount() > amount)
                 {
-                    // Deduct gold, change reputation and apply blessing
+                    // Deduct gold, and apply blessing if member
                     playerEntity.DeductGoldAmount(amount);
```

Comment:
Should the blessing be only for members? It seems to me like anyone who donates should be able to get it. I suppose restricting it gives a bit more uniqueness to joining a particular temple, but allowing it for all temples gives a reason to donate if you're not a member (I'm not aware of any benefit to raising your reputation with a temple other than for gaining ranks as a member). Also, the later games aren't a very strong source of reasons for Daggerfall IMO because game design and lore changed a good bit, but that said, in Morrowind and Oblivion you can get blessings without being a member of a religious faction.

Reply:
I think you're right now I think about it - not quite sure why I had it in my head blessings were only given to members.